mmengine/config/config.py

```diff
@@ -155,7 +155,8 @@ def __init__(self,
     @staticmethod
     def fromfile(filename: Union[str, Path],
```

Comment:
We should store the source of variables loaded from the environment as an attribute of the config, for the convenience of tracing the source data.

Reply:
Sorry for my not fully understanding this suggestion. Could you give me some examples?

Reply:
What I mean is that we should record the environment variables used by Config in the attribute of Config. For example, Config has an attribute `env_varaible`. When we define `num_classes={{'$NUM_CLASSES:80'}}`, and `NUM_CLASSES` in `os.environ` is 1, then env_varaible will be `dict(num_classes=1)`. We should also print the `env_variable` if there are some variables loaded from `os.environ`, which could help us to track the source of the data.